Headset.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Name=Headset
+Comment=Native YouTube Player
+GenericName=Headset
+Exec=Headset %U
+Icon=Headset
```

Comment:
I get the former, the actual Hedset icon. I think this might be related to where Elementary looks for icons and where electron-installer-debian places them. I have a virtual machine running Elementary. I'll take a look there at what's going on and I'll report back and see what we can do